cluster/examples/kubernetes/ceph/csi/template/cephfs/csi-cephfsplugin-provisioner-dep.yaml

```diff
@@ -120,7 +120,7 @@ spec:
       volumes:
         - name: socket-dir
           hostPath:
-            path: "{{ .KubeletDirPath }}/plugins/{{ .DriverNamePrefix }}cephfs.csi.ceph.com"
+            path: "/var/lib/csi/plugins/{{ .DriverNamePrefix }}cephfs.csi.ceph.com"
```

Comment:
Does this need to be configurable for any reason?

Reply:
if required I can  make it configurable

Reply:
I'm assuming this can be made configurable later if needed? If so, we can leave it for later.